packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -120,56 +148,32 @@ export function PanelChrome({
         ) : null}
       </div>
 
-      <div className={cx(styles.headerContainer, dragClass)} style={headerStyles} data-testid="header-container">
-        {title && (
-          <h6 title={title} className={styles.title}>
-            {title}
-          </h6>
-        )}
+      {(hoverHeader || !hasHeader) && menu && (
+        <HoverWidget menu={menu} title={title} offset={hoverHeaderOffset} dragClass={dragClass}>
+          {headerContent}
+        </HoverWidget>
+      )}
+
+      {hasHeader && (
+        <div className={cx(styles.headerContainer, dragClass)} style={headerStyles} data-testid="header-container">
+          {headerContent}
 
-        <PanelDescription description={description} className={dragClassCancel} />
+          <div className={styles.rightAligned}>
+            {menu && <PanelMenu menu={menu} title={title} menuButtonClass={cx(styles.menuItem, 'show-on-hover')} />}
```

Comment:
the menu doesn't receive `dragClassCancel` and can be used for dragging

Reply:
Ah, right, messed that up during the merge. Thanks for spotting it!

Reply:
Based on the slack conversation, we should put back the `dragClassCancel`, the side effect of dragging and opening the menu is not ideal.

https://user-images.githubusercontent.com/239999/216286790-4b177780-f3c3-4360-ae45-2930a86ca843.mp4